cpu/kinetis/dist/calc_spi_scalers/calc_spi_scalers.c

```diff
@@ -28,6 +28,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#define ARRAY_SIZE(a) (sizeof((a)) / sizeof((a)[0]))
```

Comment:
```suggestion
#ifndef ARRAY_SIZE
#define ARRAY_SIZE(a) (sizeof((a)) / sizeof((a)[0]))
#endif
```

Just in case some RIOT header might get included in the future.

Reply:
This is very unlikely to happen. This is a helper tool, completely separate from RIOT.

Reply:
> Just in case some RIOT header might get included in the future.

This will only throw an error if the macro definition is different. In which case we want the error because apparently the macro changed and silently using the new definition might result in unwanted behavior.